backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/VmCommand.java

```diff
@@ -579,6 +579,7 @@ protected boolean addVmLease(Guid leaseStorageDomainId, Guid vmId, boolean hotPl
         if (getParameters().getEntityInfo() == null) {
```

Comment:
this looks weird now, I think your change was the original change intention

Reply:
yeah, when seeing that there's that pre-polling I was looking for a place to see the entity info and saw this - it took me few moments to realize that it's not set on the right parameters.. :) I'll change that